node-drainer-module/main.go

```diff
@@ -8,7 +8,7 @@
 //
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// WITHOUT WARRANTIES OR CONDITIONS KIND, either express or implied.
```

Comment:
is this change expected?

Reply:
Nope, I was updating the docs and must have missed this one.